### PR TITLE
Saas 17 valuation sle

### PR DESCRIPTION
**Pull request by sle-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/purchase/test/average_price.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 - 
   !python {model: product.product, id: product_average_icecream}: |
```
Comment (nhomar) [inline addons/purchase/test/average_price.yml]: make the xml_id consistent with the cost asigned

File: addons/purchase/test/average_price.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@
...
     assert self.qty_available == 10.0, 'Wrong quantity in stock after first reception'
-    assert self.average_price == 60.0, 'Standard price should be the price of the first reception!'
+    assert self.standard_price == 60.0, 'Standard price should be the price of the first reception!'
```
Comment (nhomar) [inline addons/purchase/test/average_price.yml]: make the xml_id consistent with the cost asigned

File: addons/purchase/test/average_price.yml (hunk 1 above)
Comment (sle-odoo) [reply addons/purchase/test/average_price.yml]: we have a task to get rid of yml files, we'll fix this one later on

**Comment by sle-odoo** [comment]
merged at 757758624f2fd699a932d2232b51b3698a793fd5

**Pull request closed without merging**